extension/popup.html

```diff
@@ -0,0 +1,37 @@
+ <html>
+    <head>
+        <link rel="stylesheet" href="./options/fonts/material-icons/material-icons.css">
+
+        <link href="https://fonts.googleapis.com/css?family=Lato" rel="stylesheet">
+
+        <link rel="stylesheet" type="text/css" href="popup.css"/>
```

Comment:
This isn't really a problem, but just for consistency-sake better to not close these `<link>` tags. In this HTML5 context, end tags are not actually allowed for `<link>` or `<meta>` tags, however the browser should just ignore it and handle it as you would expect. This is a point of confusion for a lot of people (including me), but [this stackoverflow answer](https://stackoverflow.com/a/19510239) has a bit more info if you're interested.